contracts/src/arbitration/KlerosCoreBase.sol

```diff
@@ -99,6 +101,7 @@ abstract contract KlerosCoreBase is IArbitratorV2, Initializable, UUPSProxiable
     Dispute[] public disputes; // The disputes.
     mapping(IERC20 => CurrencyRate) public currencyRates; // The price of each token in ETH.
     bool public paused; // Whether asset withdrawals are paused.
+    address public wNative; // The address for WETH tranfers.
```

Comment:
wNative should be immutable to save gas, and set via constructor. Would save 2k gas every time it's called. Can still be upgraded as everything else in the contract

Sample:
https://github.com/greenlucid/pgtcr/blob/e92f1bbbefe003883ab7f0d5a86a75dcfe150152/contracts/src/PermanentGTCR.sol#L199